### PR TITLE
Added association alumni to groups.

### PR DESCRIPTION
**Pull request by Dr-Horv**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -20,7 +20,28 @@ class User < ActiveRecord::Base
...
+		:nollkit, 
+		:armit, 
+		:digit, 
+		:fanbarerit, 
+		:fritid, 
+		:'8bit', 
+		:drawit, 
+		:flashit, 
+		:hookit, 
+		:revisorer,
```
Comment (lindskogen) [inline app/models/user.rb:37]: Formatter for revisorer?

**Comment by kirayatail** [comment]
Saknar valberedningen

File: app/models/user.rb [hunk 2]
```diff
@@ -92,8 +113,14 @@ def itize
 				'P.R.I.T.'
 			when :fanbarerit
 				'FanbärerIT'
+			when :sprit
+				'S.P.R.I.T.'
+			when :nollqit
+				'NollQIT'
 			when :valberedningen
 				'Valberedningen'
+			when :revisorer
```
Comment (lindskogen) [inline app/models/user.rb:122]: Maybe normalize these formatters to a singular one that just titlecases the word

Comment (abjugard) [inline app/models/user.rb:122]: Or pull the actual display name value from LDAP? Could easily build a group info page into account.

**Comment by lindskogen** [comment]
@Dr-Horv status on this one?

**Comment by Dr-Horv** [comment]
@lindskogen Not sure, come to think of it maybe isn't that important?

**Comment by malmers** [comment]
This change is probably not relevant, thanks for the submission though!

**Pull request closed without merging**